python/tank/authentication/shotgun_shared/saml2_sso.py

```diff
@@ -233,7 +239,7 @@ def update_browser_from_session(self):
         qt_cookies = []
         if self._session is not None:
             cookies = _decode_cookies(self._session.cookies)
-            qt_cookies = [QtNetwork.QNetworkCookie.parseCookies(cookie)[0] for cookie in cookies]
+            qt_cookies = QtNetwork.QNetworkCookie.parseCookies(cookies.output(header=''))
```

Comment:
are we clear at this point QT wise? Could `QtNetwork` ever be `None` at this point? Please add a comment to explain. And potentially a try/except if we need to test.